### DISCUSSION TITLE
Slow local TX confirmation

### DISCUSSION THREAD

**Discussion by zezic** [phantom/docs#161] [Help & Support] [answered]
### Summary

When developing Solana apps using local validator and signing transactions with Phantom it sends (failing) requests like these two:
 
1. https://api.phantom.app/transaction/scan/v1?language=en-US
2. https://api.phantom.app/image-proxy/?image=data%3Aimage%2 [...] long part skipped
 
First type of requests (TX scan) fails with the message:
```json
{
    "message":"A validation error has occured",
    "errors": {
        "chainIds":["each value in chainIds must be a valid enum value"]
    }
}
```
 
Second type of requests (icon resize) fails with 500 error.
 
All this makes the local app development very painful because it makes Phantom wait for every signature confirmation for too long because of the errors and when it finally shows the confirmation button it leaves the time window which only lasts for about 1 or 2 seconds (you better be pretty quick to click on "Confirm" button!) and then it starts to spin the spinner again when doing all these failing requests in the background.
 
Please, help us to disable the TX scan for local validator and allow to disable icon resize procedure using Phantom remote server.

### Steps to Reproduce

1. Deploy your unique Solana program on a local hosted test validator (`solana-test-validator`)
2. Serve the web app on the `localhost:3000` URL (the page should have the favicon for better effect)
3. Try to sign any transaction for your program using Phantom with "Testnet Mode" set to "Solana Localnet"
4. Experience abnormally long loading spinner and failing network requests in addon network monitor

### Phantom Version

23.18.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hey @zezic,

Thank you for sending in this feedback. We do have an internal ticket to address this but it has not yet been prioritized. Please do keep an eye on our release notes for updates. Additionally feel free to add this to our [feature request board](https://phantom.canny.io).

**Reply by zezic** [reply, marked as answer]
OK! I added tickets to the feature request board. These will help Solana-app devs alot because Phantom is most popular software Solana wallet and it's often used by developers to test their apps on local machines.

**Reply by adamdelphantom** [reply]
Thank you!

**Comment by hanool** [comment]
Any progress?
